Package.swift

```diff
@@ -1386,7 +1405,7 @@ func googleAppMeasurementDependency() -> Package.Dependency {
 
   // Point SPM CI to the tip of main of https://github.com/google/GoogleAppMeasurement so that the
   // release process can defer publishing the GoogleAppMeasurement tag until after testing.
-  if ProcessInfo.processInfo.environment["FIREBASECI_USE_LATEST_GOOGLEAPPMEASUREMENT"] != nil {
+  if Context.environment["FIREBASECI_USE_LATEST_GOOGLEAPPMEASUREMENT"] != nil {
```

Comment:
This is an unrelated to the Swift tools version bump. This API was added to the `PackageDescription` module in Swift 5.6, and seems slightly cleaner (e.g. importing ProcessInfo is unneeded).